lib/src/pages/layouts/yaru_landscape_layout.dart

```diff
@@ -63,51 +106,136 @@ class _YaruLandscapeLayoutState extends State<YaruLandscapeLayout> {
+            : MouseCursor.defer,
+        child: child,
+      );
+    }
+
+    return child;
+  }
+
+  Widget _buildLeftPane() {
+    return Container(
+      width: _leftPaneWidth,
+      decoration: BoxDecoration(
+        border: Border(
+          right: BorderSide(
+            width: 1,
+            color: _separatorColor,
           ),
         ),
-        Expanded(
-          child: Theme(
-            data: Theme.of(context).copyWith(
-              pageTransitionsTheme: theme.landscapeTransitions,
-            ),
-            child: Navigator(
-              pages: [
-                MaterialPage(
-                  key: ValueKey(_selectedIndex),
-                  child: widget.length > _selectedIndex
-                      ? widget.pageBuilder(context, _selectedIndex)
-                      : widget.pageBuilder(context, 0),
-                ),
-              ],
-              onPopPage: (route, result) => route.didPop(result),
-              observers: [HeroController()],
-            ),
+      ),
+      child: Scaffold(
+        appBar: widget.appBar,
+        body: YaruPageItemListView(
+          length: widget.length,
+          selectedIndex: _selectedIndex,
+          onTap: _onTap,
+          builder: widget.tileBuilder,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildPage(BuildContext context) {
+    final theme = YaruMasterDetailTheme.of(context);
+
+    return Theme(
+      data: Theme.of(context).copyWith(
+        pageTransitionsTheme: theme.landscapeTransitions,
+      ),
+      child: Navigator(
+        pages: [
+          MaterialPage(
+            key: ValueKey(_selectedIndex),
+            child: widget.length > _selectedIndex
+                ? widget.pageBuilder(context, _selectedIndex)
+                : widget.pageBuilder(context, 0),
+          ),
+        ],
+        onPopPage: (route, result) => route.didPop(result),
+        observers: [HeroController()],
+      ),
+    );
+  }
+
+  Widget _buildLeftPaneResizer(BuildContext context) {
+    return Positioned(
+      child: AnimatedContainer(
+        duration: _kLeftPaneResizingRegionAnimationDuration,
+        color:
+            _isHovering || _isDragging ? _separatorColor : Colors.transparent,
+        child: MouseRegion(
+          cursor: SystemMouseCursors.resizeColumn,
+          onEnter: (event) => setState(() {
+            _isHovering = true;
+          }),
+          onExit: (event) => setState(() {
+            _isHovering = false;
+          }),
+          child: GestureDetector(
+            onPanStart: (details) => setState(() {
+              _isDragging = true;
+              _initialPaneWidth = _leftPaneWidth;
+            }),
+            onPanUpdate: (details) => setState(() {
+              _paneWidthMove += details.delta.dx;
+              final width = _initialPaneWidth + _paneWidthMove;
+
+              final previousPaneWidth = _leftPaneWidth;
+              final contextWidth = MediaQuery.of(context).size.width;
```

Comment:
this assumes that the layout covers the whole window?

Reply:
maybe LayoutBuilder could be used to get the actual available width?

Reply:
Oh, I didn't know this.